Formula/survex.rb

```diff
@@ -0,0 +1,72 @@
+class Survex < Formula
+  desc "Cave Surveying Tool"
+  homepage "https://www.survex.com"
+  url "https://survex.com/software/1.2.32/survex-1.2.32.tar.gz"
+  sha256 "ac15252618f6a59617ec1142609f04bba69b1f18b3490729e405ff89a4b22852"
+
+  depends_on "wxmac"
+  depends_on "proj"
+  depends_on "ffmpeg"
+
+  depends_on "gettext" => :build
+
+  def install
+    # Allow for non-static linking against PROJ EPSG/ESRI definitions
+    ["src/cavern.c", "src/aven.cc"].each { |f| inreplace f, /pj_set_finder\(msg_proj_finder\)/, "" }
+
+    # Prevent assumption that this is a MacOS bundle build
+    inreplace "src/message.c", /#ifdef MACOSX_BUNDLE/, "#if 0"
+
+    system "./configure", "--prefix=#{prefix}",
+                          "--bindir=#{bin}",
+                          "--mandir=#{man}",
+                          "--docdir=#{doc}",
+                          "--datadir=#{share}"
+
+    system "make"
+    system "make", "install"
+
+    # Create and populate Aven.app
+    mkdir_p ["Aven.app/Contents/MacOS", "Aven.app/Contents/Resources"]
```

Comment:
Same here: the package should be built from a makefile or a script, but we're not going to populate it ourselves in the formula. This is too fragile, likely to break at any point.